test/core/modules/webgl-distort.js

```diff
@@ -0,0 +1,49 @@
+var test = require('tape');
+var base64Img = require('base64-img');
+var looksSame = require('looks-same');
+
+require('../../../src/ImageSequencer.js');
+
+var sequencer = ImageSequencer({ ui: false });
+var options = {
+        nw: '0 100',
+        ne: '1023 50',
+        se: '1223 867',
+        sw: '100 767'
+    };
+var target = 'test_outputs';
+var red = "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQAQMAAAAlPW0iAAAABGdBTUEAALGPC/xhBQAAACBjSFJNAAB6JgAAgIQAAPoAAACA6AAAdTAAAOpgAAA6mAAAF3CculE8AAAABlBMVEX+AAD///+KQee0AAAAAWJLR0QB/wIt3gAAAAd0SU1FB+EGHRIVAvrm6EMAAAAMSURBVAjXY2AgDQAAADAAAceqhY4AAAAldEVYdGRhdGU6Y3JlYXRlADIwMTctMDYtMjlUMTg6MjE6MDIrMDI6MDDGD83DAAAAJXRFWHRkYXRlOm1vZGlmeQAyMDE3LTA2LTI5VDE4OjIxOjAyKzAyOjAwt1J1fwAAAABJRU5ErkJggg==";
+var benchmark = "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAH0lEQVQ4T2NkoBAwUqifYdQAhtEwYBgNA1A+Gvi8AAAmmAARf9qcXAAAAABJRU5ErkJggg==";
```

Comment:
Hi! Hmm, not sure; is this supposed to show anything? I tried opening it in a browser but don't see an image.

Reply:
I do see a small grey-ish 16x16 pixel img.

Reply:
I'll check it out!